ballerina/Dependencies.toml

```diff
@@ -5,12 +5,12 @@
 
 [ballerina]
 dependencies-toml-version = "2"
-distribution-version = "2201.11.0"
+distribution-version = "2201.12.10"
```

Comment:
any particular reason to bump the minimum compatible ballerina version along with these keyword changes?